cypress/integration/capture.spec.js

```diff
@@ -194,6 +196,74 @@ describe('Event capture', () => {
         })
     })
 
+    describe('capturing performance', () => {
+        it('sends performance timing with the pageview when enabled', () => {
+            given('options', () => ({ capture_pageview: true, capture_performance: true }))
+            start()
+
+            // Pageview will be sent immediately
+            cy.wait('@capture').its('request.headers').should('deep.equal', {
+                'Content-Type': 'application/x-www-form-urlencoded',
+            })
+            cy.get('@capture').should(({ request }) => {
+                const captures = getBase64EncodedPayload(request)
+
+                expect(captures.event).to.equal('$pageview')
+
+                const performance = captures.properties.performance
+
+                /**
+                 * The performance propery holds three items.
+                 * The result of asking window.performance for entries by type of navigation, resource, and paint
+                 * We cannot guarantee what is available in resource and paint when it runs
+                 *
+                 * Navigation will be a list with exactly one `PerformanceEntry` object
+                 *
+                 * Each of resource and paint will be a list with zero or more `PerformanceEntry` objects
+                 *
+                 * https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByType
+                 */
+
+                expect(performance).to.have.property('navigation')
+                expect(performance.navigation).to.be.instanceof(Array).and.to.have.length(1)
+                expect(performance.navigation[0]).to.have.property('domContentLoadedEventEnd')
+                expect(performance.navigation[0].domContentLoadedEventEnd).to.be.greaterThan(0)
+
+                expect(performance).to.have.property('resource')
+                expect(performance.resource).to.be.instanceof(Array).and.to.have.length.greaterThan(0)
+                expect(performance.resource[0]).to.have.property('entryType', 'resource')
+                expect(performance.resource[0]).to.have.property('connectEnd')
+
+                expect(performance).to.have.property('paint')
+                expect(performance.paint).to.be.instanceof(Array)
+                if (performance.paint.length > 0) {
+                    // we can't guarantee we run early enough to capture paint results
+                    performance.paint.forEach((paintResult) => {
+                        expect(paintResult).to.have.property('startTime')
+                        expect(paintResult.startTime).to.be.greaterThan(0)
+                    })
+                }
+            })
+        })
+
+        it('does not send performance timing with the pageview when disabled', () => {
```

Comment:
Is this a useful test? I don't think it is - we're asserting the same behavior in unit tests, browser context gives us nothing extra.